File: CSharp/area-triangulo/area-triangulo/entities/Triangulo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace area_triangulo.entities
{
    internal class Triangulo
    {
        private double ladoA;
        private double ladoB;
        private double ladoC;

        public Triangulo(double ladoA, double ladoB, double ladoC)
        {
            this.ladoA = ladoA;
            this.ladoB = ladoB;
            this.ladoC = ladoC;
        }

        public double calcularArea()
        {
            double p = (ladoA + ladoB + ladoC) / 2;
            return Math.Sqrt(p * (p - ladoA) * (p - ladoB) * (p - ladoC));
        }
    }
}

File: Java/area-triangulos/src/entities/Triangulo.java
package entities;

public class Triangulo {
	double ladoA;
	double ladoB;
	double ladoC;
	
	public Triangulo(double ladoA, double ladoB, double ladoC) {
		this.ladoA = ladoA;
		this.ladoB = ladoB;
		this.ladoC = ladoC;
	}
	
	public double calcularArea() {
		double p = (this.ladoA + this.ladoB + this.ladoC)/2;
		return Math.sqrt(p * (p - this.ladoA) * (p - this.ladoB) * (p - this.ladoC));
	}
}
